### PR TITLE
Publish RubyGems Monthly Updates for March 2023

### PR DESCRIPTION
**Pull request by lauragift21**
RubyGems update roundup for March 2023.

Hi @deivid-rodriguez, can you help review this month's update to ensure I accurately captured the work done last month?

### CODE REVIEW

**Comment by mensfeld** [comment]
Shouldn't we mention the browning / reaction to it on our API? I'm not sure where we stand with this but it did cause some havoc ;)

**Comment by lauragift21** [comment]
> Shouldn't we mention the browning / reaction to it on our API? I'm not sure where we stand with this but it did cause some havoc ;)

Sure, I'm happy to include it, I will be curious to hear what @indirect or @deivid-rodriguez thoughts are on this.

**Review by deivid-rodriguez — COMMENTED** [review]
I'd group the client side improvements into:

* Enhance dealing with tar files to support future server side gem content navigation feature (https://github.com/rubygems/rubygems/pull/6461, https://github.com/rubygems/rubygems/pull/6476).
* Improved auto-healing of corrupted lockfiles and fixed some related regressions (https://github.com/rubygems/rubygems/pull/6400, https://github.com/rubygems/rubygems/pull/6423, https://github.com/rubygems/rubygems/pull/6552, https://github.com/rubygems/rubygems/pull/6540, https://github.com/rubygems/rubygems/pull/6532, https://github.com/rubygems/rubygems/pull/6495).
* Fix some resolution edge cases (https://github.com/rubygems/rubygems/pull/6330, https://github.com/rubygems/rubygems/pull/6442, https://github.com/rubygems/rubygems/pull/6441, https://github.com/rubygems/rubygems/pull/6535

Even if not end user visible improvements, I'd also mention @hsbt's efforts to unify RubyGems & Bundler codebase.

And even if not yet release, I think it's also worth mentioning efforts to support WebAuthn from the CLI (https://github.com/rubygems/rubygems/pull/6523).

The other improvements included in the initial list (https://github.com/rubygems/rubygems/pull/6413, https://github.com/rubygems/rubygems/pull/6537, https://github.com/rubygems/rubygems/pull/6428, https://github.com/rubygems/rubygems/pull/6461), I'd probably move them to a "Other improvements" section.

Regarding the dependency API brown out, I do think it's worth mentioning it in the rubygems.org news.

File: _posts/2023-04-05-march-rubygems-updates copy.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+- removed delayed job `statsd` deployment since it's no longer in use - [#3642](https://github.com/rubygems/rubygems.org/pull/3642).
+- stored and yanked gem contents in background jobs - [#3454](https://github.com/rubygems/rubygems.org/pull/3454).
+- responded to CLI on webauthn verification - [#3535](https://github.com/rubygems/rubygems.org/pull/3535).
+
+In March, RubyGems.org gained [193 new commits](https://github.com/rubygems/rubygems.org/compare/master@%7B2023-03-01%7D...master@%7B2023-03-31%7D), contributed by 15 authors. There were 7,462 additions and 2,106 deletions across 337 files.
+
+## Thank you
+Thank you to all the contributors of RubyGems and RubyGems.org for this month! Your contributions are greatly appreciated, and we are grateful for your support.
+
+### Contributors to RubyGems:
```
Comment (simi) [inline _posts/2023-04-05-march-rubygems-updates copy.md:47]: _missing newline after title_

Comment (lauragift21) [reply _posts/2023-04-05-march-rubygems-updates copy.md:47]: Thanks fixed!

**Comment by lauragift21** [comment]
> dependency API brown out

Hi @mensfeld, This was covered in the previous RubyGems.org update: https://blog.rubygems.org/2023/03/11/february-rubygems-updates.html

**Comment by mensfeld** [comment]
@lauragift21 thank you :pray:

**Pull request merged**